### ISSUE TITLE
Clear storage tip: show direct button to start app info page

### ISSUE THREAD

**Issue by q-wertz** [stefan-niedermann/nextcloud-deck#456]
**Describe the bug**
Only a cosmetic bug...
The explanation how to delete the storage of the Nextcloud / Nextcloud Deck app does not really fit the Android naming (at least on Android 10)
https://github.com/stefan-niedermann/nextcloud-deck/blob/c45bb3ce09f5bf7aab80f89aba047c32f103b741/app/src/main/res/values/strings.xml#L231

**Expected behavior**
On Android 10 (maybe bad translated from German, maybe someone can give the official naming) it is:
Apps & Notifications → Show all X apps →  Nextcloud / Nextcloud Deck → Storage and Cache → Delete X 
(here it's possible to select between Cache and storage. sure you need storage and cache is not sufficient?)

**Smartphone (please complete the following information):**
- Device: Nokia 7 plus
- Android-Version: (e.g. 10)

**Comment by stefan-niedermann** [comment]
Yep, sure it's `storage` and not `cache`.

Hmm. I don't think there is a general description that fits to every device, rom and version...

**Comment by stefan-niedermann** [comment]
Maybe we can simply use a button and open this page directly on click 🤔

**Comment by desperateCoder** [comment]
Nope, waaay too good ux. 😉

**Comment by stefan-niedermann** [comment]
Shut up, backend donkey!

**Comment by desperateCoder** [comment]
❤️

**Comment by stefan-niedermann** [comment]
❤️

**Comment by stefan-niedermann** [comment]
Found [this information](https://stackoverflow.com/a/4772481), should be possible:

```java
private static final String SCHEME = "package";

private static final String APP_PKG_NAME_21 = "com.android.settings.ApplicationPkgName";

private static final String APP_PKG_NAME_22 = "pkg";

private static final String APP_DETAILS_PACKAGE_NAME = "com.android.settings";

private static final String APP_DETAILS_CLASS_NAME = "com.android.settings.InstalledAppDetails";

public static void showInstalledAppDetails(Context context, String packageName) {
    Intent intent = new Intent();
    final int apiLevel = Build.VERSION.SDK_INT;
    if (apiLevel >= 9) { // above 2.3
        intent.setAction(Settings.ACTION_APPLICATION_DETAILS_SETTINGS);
        Uri uri = Uri.fromParts(SCHEME, packageName, null);
        intent.setData(uri);
    } else { // below 2.3
        final String appPkgName = (apiLevel == 8 ? APP_PKG_NAME_22
                : APP_PKG_NAME_21);
        intent.setAction(Intent.ACTION_VIEW);
        intent.setClassName(APP_DETAILS_PACKAGE_NAME,
                APP_DETAILS_CLASS_NAME);
        intent.putExtra(appPkgName, packageName);
    }
    context.startActivity(intent);
}
```

**Comment by q-wertz** [comment]
> Hmm. I don't think there is a general description that fits to every device, rom and version...

Me neither. I think the "navigation description" is just a little bit misleading for non technical users.
The button idea sounds quite nice to me.

**Comment by stefan-niedermann** [comment]
jup. will implement it that way. Something like

```
💡  Clear the storage of the app in the app info screen.

                  [ APP INFO SCREEN ]
```

**Comment by stefan-niedermann** [comment]
Everything is read, just waiting for a change in the Single-Sign-On library... https://github.com/nextcloud/Android-SingleSignOn/pull/203

**Comment by stefan-niedermann** [comment]
Fixed in `1.4.4`.
